recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -141,6 +141,27 @@ make_partitions() {
     return 0
 }
 
+create_filesystems() {
+    local partnumber
+    local pline mke2fscmd fstype
+		local errlog=$(mktemp)
+    for pline in "${PARTS[@]}"; do
+			eval "$pline"
+	    if [ -n "$fstype" ] && [ "$fstype" != "basic" ]; then
+				printf "Creating $fstype filesystem to /dev/$DEVNAME$PARTSEP$partnumber"
+				$mke2fscmd="mke2fs -t $fstype /dev/$DEVNAME$PARTSEP$partnumber"
```

Comment:
It might be better to use `mkfs.$fstype` here, to allow for other filesystem types than ext2/3/4.